docs/contributing.rst

```diff
@@ -6,6 +6,15 @@ Thank you for your interest in contributing! We welcome all contributions no mat
 Setting the stage
 ~~~~~~~~~~~~~~~~~
 
+.. note::
+    If you are on macOS and it is the first time you install Py-EVM, you will need to install LevelDB with brew:
+
+    .. code:: sh
+        brew install python3 LevelDB
```

Comment:
Maybe just add `LevelDB` to this section: https://py-evm.readthedocs.io/en/latest/guides/quickstart.html#installing-on-macos

Maybe it could make sense to link from here to that section, though.

Reply:
Done!